packages/blockchain-wallet-v4/src/signer/btc.js

```diff
@@ -44,8 +44,9 @@ export const signWithWIF = curry((network, selection) =>
   compose(signSelection(network), wifToKeys(network))(selection)
 )
 
-export const signMessage = (priv, message) => {
+export const signMessage = (priv, addr, message) => {
   const keyPair = privateKeyStringToKey(priv, 'base58')
```

Comment:
Can you move this to privateKeyStringToKey method, because that's used by show private key too, and I think we should be displaying the uncompressed key there too.

Reply:
Good point, will do